cmd/config/get_test.go

```diff
@@ -72,7 +78,45 @@ func (suite *ConfigGetTestSuite) TestConfigGetUserRole() {
 	}
 }
 
+func (suite *ConfigGetTestSuite) TestConfigGetTenants() {
+	getEnvironmentsExpected := []configClient.Environment{{
+		Name: "testTenant",
+	}}
+	var getRolesExpected []model.RoleConfig
+
+	if err := registerResponder(getEnvironmentsExpected, http.StatusOK, "/environments", http.MethodGet); err != nil {
+		suite.T().Fatal(err)
+	}
+
+	err := registerResponder(getRolesExpected, http.StatusOK, "/roles", http.MethodGet)
+	if err != nil {
+		suite.T().Fatalf("TestDeployStartJsonSuccess failed with: %s", err)
```

Comment:
I'm not sure why every log line in this file says "TestDeployStartJsonSuccess", but you certainly don't have to follow that pattern 😄 

If you call Fatalf, I'm pretty sure the `testing` package gives you pretty good clues about which test called it - you don't need to include the name of the test in the log.

Reply:
In my opinion you should not `Fatalf` here at all. 

In most cases I'd just do `assert.NoError(t, registerResponder(getRolesExpected, http.StatusOK, "/roles", http.MethodGet))`

See https://github.com/golang/go/wiki/TestComments#keep-going

Reply:
Thanks @danielpeach this is pretty useful, I just clean up most that I could, that was a very good reading